src/main.cpp

```diff
@@ -51,7 +51,8 @@ int main( int argc, char **argv )
 	}
 #endif
 
-	QApplication::setAttribute( Qt::AA_DisableWindowContextHelpButton );
+	// AA_DisableWindowContextHelpButton
+	// QApplication::setAttribute( Qt::AA_DisableWindowContextHelpButton  );
```

Comment:
This is required on windows.

Reply:
And no longer exists in QT 6

Reply:
If it no longer exists, why not remove it?

Reply:
forgor :skull: